# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/pyslsk/typo.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur pyslsk-1.2.3.orig/pyslsk pyslsk-1.2.3/pyslsk
--- ./pyslsk	2003-06-23 19:24:12.000000000 +0200
+++ ./pyslsk	2003-08-04 02:45:19.000000000 +0200
@@ -55,7 +55,7 @@
 
 def usage():
     print """Usage: pyslsk [OPTION]...
-  -c file,	--config=file	Use non-defult configuration file
+  -c file,	--config=file	Use non-default configuration file
   -h,		--help		Display this help and exit
 
 Report bugs to <ak@sensi.org>""" 	
